app/assets/stylesheets/app/transaction_search.scss

```diff
@@ -35,3 +35,13 @@
     text-align: right;
   }
 }
+
+#zero_dollar_orders {
+  align-items: center;
+  display: flex;
+
+  .checkbox {
```

Comment:
I'm not sure how to make it better, but this feels overly specific, especially now that we're in full control of the partial. Is there any way we can clean it up? I would be ok if we did something with flexbox and abandoned the bootstrap grid here (flexbox wasn't a thing when nucore was first designed).